tests/test_messaging.py

```diff
@@ -1641,7 +1642,8 @@ def test_send(self):
         assert recorder[0].url == self._get_url('explicit-project-id')
         assert recorder[0].headers['X-GOOG-API-FORMAT-VERSION'] == '2'
         assert recorder[0].headers['X-FIREBASE-CLIENT'] == self._CLIENT_VERSION
-        assert recorder[0]._extra_kwargs['timeout'] is None
+        assert recorder[0]._extra_kwargs['timeout'] == pytest.approx(
```

Comment:
(again, though if you remove these assertions you should probably add a `test_timeout` test.)

Reply:
Moved to TestTimeout class